.azure-pipelines.yml

```diff
@@ -79,3 +79,9 @@ jobs:
       python setup.py check --restructuredtext --strict
       rst2html.py --halt=2 README.rst >/dev/null
     displayName: 'rst2html'
+
+  - script: |
+      pip install setuptools twine
+      python setup.py sdist bdist_wheel
+      twine check dist/*
+    displayName: 'Twine check'
```

Comment:
Could this be a separate job? It would only need to happen for one python version, probably 3.8.

Also can the `python setup.py check --restructuredtext --strict` script task be removed from the pytest job, since this covers that?

Reply:
Separate job is bad, since we would need to setup Python again, install the dependencies etc.

I added a condition for this task to only be run if Python==3.8

Reply:
Dependencies aren't required to build the package or run twine though?

Reply:
Yeah, I don't know what I was thinking when I wrote this :)

I still think that the current solution of just running it for the latest Python version in the same job is 110% fine and the best solution. The condition is simple and readable.
1. No need to setup Python nor install wheels again.
2. It's weird when the twine check is the only check that has its own job.

Reply:
The black check should have it's own job too, but that will happen separately. It's already been done this way for `anndata`.

Reply:
Yeah, we can extract all simple static analysis tasks into their own job.
However, this should be done in another PR. Are you fine with this one merged?